hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ReconTaskController.java

```diff
@@ -41,9 +41,11 @@ void consumeOMEvents(OMUpdateEventBatch events,
 
   /**
    * Pass on the handle to a new OM DB instance to the registered tasks.
+   *
    * @param omMetadataManager OM Metadata Manager instance
+   * @param reconOmTaskMap
    */
-  void reInitializeTasks(ReconOMMetadataManager omMetadataManager);
+  void reInitializeTasks(ReconOMMetadataManager omMetadataManager, Map<String, ReconOmTask> reconOmTaskMap);
```

Comment:
Please update the javadoc, what the new `reconOmTaskMap` does and what happens if it's null, etc.